pkg/schedule/checker/replica_strategy.go

```diff
@@ -98,7 +98,12 @@ func (s *ReplicaStrategy) SelectStoreToFix(coLocationStores []*core.StoreInfo, o
 	}
 	// trick to avoid creating a slice with `old` removed.
 	s.swapStoreToFirst(coLocationStores, old)
-	return s.SelectStoreToAdd(coLocationStores[1:])
+	// If the coLocationStores only has one store, no need to remove.
```

Comment:
In issue desc, there are 3 nodes labeled `disk_type=mix`.  Does it affect 3 nodes case?